tools/wpt/testfiles.py

```diff
@@ -35,6 +35,8 @@
     from typing import Text
     from typing import Tuple
 
+DEFAULT_IGNORE_RULERS = ("resources/testharness*", "resources/testdriver*")
```

Comment:
For my own info : this won't catch something like `a/b/resources/testdriver-special.js` right?

Reply:
Correct. The rules are eventually converted to regexes and `match` is be used instead of `search`; in other words, the regex must match from the beginning of a string. Hopefully the help message ("Rules are paths relative to the test root") also reflects this.